packages/vendure-plugin-accept-blue/src/api/accept-blue-common-resolvers.ts

```diff
@@ -65,19 +74,26 @@ export class AcceptBlueCommonResolver {
     @Ctx() ctx: RequestContext,
```

Comment:
Not part of this PR, but it says `stripeSubscriptions` instead of `acceptBlueSubscriptions`